predicators/approaches/active_sampler_learning_approach.py

```diff
@@ -115,6 +116,25 @@ def _update_sampler_data(self) -> None:
                 ns = segment.states[-1]
                 success = self._check_option_success(o, segment)
 
+                if CFG.env == "bumpy_cover" and CFG.bumpy_cover_right_targets:
```

Comment:
can we put this logic behind a flag so we can easily compare to the old version too? for example, rather than `myopic_classifier`, something like `teacher_classifier`?

Reply:
also behind this flag let's assert that the env is `bumpy_cover`